### DISCUSSION TITLE
Disable casks

### DISCUSSION THREAD

**Discussion by quackerex** [Homebrew/discussions#5597] [Casks] [answered]
### Output of `brew config`

```shell
N/A
```

### Output of `brew doctor`

```shell
N/A
```

### Description of issue

I don't use casks and I noticed that brew check for new casks every time I do an update. Is it possible to disable casks completely?

### Relevant casks

N/A

**Comment by daeho-ro** [comment]
Why not just uninstall?

**Comment by SMillerDev** [comment, marked as answer]
No, this is not currently supported
